Admin/AdminInterface.php

```diff
@@ -937,6 +937,7 @@ public function getLabel();
     public function getPersistentParameters();
 
     /**
+     * NEXT MAJOR: remove this signature
```

Comment:
An interface is a contract between classes. Remove a signature is changing the contract, and client code relying on this interface will have to change to no longer call this method.